argus/backend/service/jenkins_service.py

```diff
@@ -220,7 +220,9 @@ def clone_build_job(self, build_id: str, params: dict[str, str]):
     def build_job(self, build_id: str, params: dict, user_override: str = None):
         queue_number = self._jenkins.build_job(build_id, {
             **params,
-            self.RESERVED_PARAMETER_NAME: g.user.username if not user_override else user_override
+            # use the user's email as the default value for the requested by user parameter,
+            # so it would align with how SCT default works, on runs not trigger by argus
+            self.RESERVED_PARAMETER_NAME: g.user.email.split('@')[0] if not user_override else user_override
```

Comment:
Could it be that there are users that do not expose their emails?
maybe we should have a fallback to username if we couldn't get email.

Reply:
The users that hide their email will usually have a github bounce email, so that can be used as a fallback (or first email available as a fallback)

Reply:
I did a fallback to the email mentioned as primary, if couldn't find a scylla email

anyhow I'm good with people that have not email configured, they won't be able to rebuild.
i.e. no name for list on resources, no runs for you...

they can fix that on their own, and set the email in user preferences.

Reply:
The question is what error they will have and if it'll be clear enough for them to understand that.
I'm pretty sure there are many users without any email exposed.

Reply:
it can easily check, if we have such user in our DB...